src/index.html

```diff
@@ -17,6 +29,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="moyo_header">
+      <div class="moyo_header_logo">
+        <a
+          href="#"
+          class="menu_link"
+        >
+          <img
+            src="images/logo.png"
+            alt="Logo"
+          />
+        </a>
+      </div>
+      <nav>
+        <ul class="moyo_menu">
+          <li class="menu-item">
+            <a
+              href="#"
+              class="menu_link is-active"
+            >
```

Comment:
Don't use spaces in `<a>` tag's `href` property. Anchor links should start with the `#` symbol followed by a valid id or be a valid URL.